src/kaltura-player.ts

```diff
@@ -832,15 +836,17 @@ export class KalturaPlayer extends FakeEventTarget {
   }
 
   private _addAudioPreset(options: Partial<KalturaPlayerConfig>): void {
-    if (options.plugins?.audioPlayer) {
-      if (!window.kalturaCustomPresetMap) {
-        window.kalturaCustomPresetMap = {};
-      }
-      if (!window.kalturaCustomPresetMap[options.ui!.targetId]) {
-        window.kalturaCustomPresetMap[options.ui!.targetId] = {};
+    Object.keys(options.plugins).forEach((plugin) => {
+      if (window.kalturaCustomPreset[plugin]) {
+        if (!window.kalturaCustomPresetMap) {
+          window.kalturaCustomPresetMap = {};
+        }
+        if (!window.kalturaCustomPresetMap[options.ui!.targetId]) {
+          window.kalturaCustomPresetMap[options.ui!.targetId] = {};
+        }
+        window.kalturaCustomPresetMap[options.ui!.targetId][plugin] = window.kalturaCustomPreset[plugin];
```

Comment:
won't this put window.kalturaCustomPreset[plugin] into every player on the window, which is what you want to avoid doing ?

Reply:
no, the idea is to add the preset in the audio plugin into "audioPlayer" key so we don't need to search for specific plugin, this is also for future cases that we will add custom preset on on another places.
It's Moshe change